tests/Functional/Controller/CRUDControllerTest.php

```diff
@@ -34,8 +34,9 @@ public function testList(): void
 
     public function testCreate(): void
     {
+        $subclass = uniqid('subclass');
         $client = static::createClient();
-        $crawler = $client->request(Request::METHOD_GET, '/admin/tests/app/foo/create');
+        $crawler = $client->request(Request::METHOD_GET, '/admin/tests/app/foo/create?subclass='.$subclass);
```

Comment:
IMHO this should go in a new test instead of modifying this one

Reply:
Indeed @mpoiriert, two tests are better than one

Reply:
Done